infrastructure/main.tf

```diff
@@ -2,11 +2,14 @@ provider "azurerm" {}
 
 module "send-letter-producer-service" {
   source   = "git@github.com:contino/moj-module-webapp?ref=master"
-  product  = "${var.product}-api"
+  product  = "${var.product}-producer"
```

Comment:
could we not have the `producer` suffix in the variable itself ?
`default = "send-letter-producer"

Reply:
That would be product+component. We need to know the product, because we'll have another component here (coming in the next pull request) - the queue, named `${var.product}-servicebus-queue`